supervisor/resolution/fixups/addon_execute_repair.py

```diff
@@ -17,6 +18,8 @@ def setup(coresys: CoreSys) -> FixupBase:
 class FixupAddonExecuteRepair(FixupBase):
     """Storage class for fixup."""
 
+    attempts = 0
```

Comment:
I think this should not be a class variable, so:

```suggestion
    def __init__(self, coresys: CoreSys) -> None:
        """Initialize the add-on execute repair fixup class."""
        super().__init__(coresys)
        self.attempts = 0
```